projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -265,6 +265,7 @@ if (environment.requestedDeliveryDate) {
     // CXSPA-6793: refactor to`provideFeatureToggles` and `satisfies` keyword
     provideFeatureTogglesFactory(() => {
       const appFeatureToggles: Required<FeatureToggles> = {
+        showProposalDownloadButton: false,
```

Comment:
Why not setting this to true? This is for our internal environment (and for standard quotes that does not harm, in case no attachments present, nothing will be displayed)

Reply:
For someone using it with older commerce (eg 2211.20) the UI would still check for `sapAttachments` in the response while getting details of a quote. 
No sure if this could break the app, but setting the feature to `false` by default is a safe option IMO.

Reply:
ok, your call!